dpc-go/dpc-api/src/service/public/router.go

```diff
@@ -20,6 +22,12 @@ func buildPublicRoutes(cont controllers) http.Handler {
 	r.With(middleware2.Sanitize).Route("/v2", func(r chi.Router) {
 		r.Use(middleware.SetHeader("Content-Type", "application/fhir+json; charset=UTF-8"))
 		r.Get("/metadata", cont.Metadata.Read)
+		r.Get("/_health", func(w http.ResponseWriter, r *http.Request) {
+			m := make(map[string]string)
+			m["database"] = "ok"
```

Comment:
Hmm. It doesn't look like we actually contact the database here. I know historically we've done database and BFD health checks, but are we limited to those options? We're simply saying as an API, we're ping-able.

If we're going to say the database connection is ok, let's test it.

Reply:
Attribution is the only thing that uses the database. I think the go app doesn’t start if it can’t connect to the database. So should we just ping attribution to see if it works?

Reply:
This is a statement that the API service _currently_ thinks it's healthy. I'd be happy enough with:
```suggestion
			m["api"] = "ok"
```